samples/interactive-tutorials/events/import-user-events-big-query.js

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(datasetId) {
+  // [START retail_import_user_events_big_query]
+
+  // Imports the Google Cloud client library.
+  const {UserEventServiceClient} = require('@google-cloud/retail').v2;
+
+  const projectNumber = process.env['GCLOUD_PROJECT'];
```

Comment:
I'd rather not rely on env variables if possible, especially for running tests locally. If your tutorial relies on it, it's fine but I'd rather users have to specifically change these variables.